### PR TITLE
Close main navigation dropdown menus on touchscreen

### PR DESCRIPTION
**Pull request by trenzterra**
Closes dropdown whenever you tap on anywhere else except the dropdown menu

### CODE REVIEW

File: js/functions.js [hunk 1]
```diff
@@ -79,6 +79,12 @@
 		// Toggle `focus` class to allow submenu access on tablets.
 		function toggleFocusClassTouchScreen() {
 			if ( window.innerWidth > 910 ) {
+				$( 'body' ).on( 'touchstart.twentysixteen', function( ) {
+					if ( ! $( event.target ).closest( '.main-navigation li' ).length )
+					{
```
Comment (iamtakashi) [inline js/functions.js]: Before testing this further, can you move this curly brace to the same line as the `if` check for the sake of consistency?

**Comment by trenzterra** [comment]
Changed.

**Comment by iamtakashi** [comment]
It works in Chrome, Safari, and Opera but It doesn't seem to work as intended in Firefox (v41) in Android (v5.1.1). However, I guess that's not a blocker for merge because it's a "nice-to-have" enhancement.

@trenzterra, what's your WordPress.org username?

**Comment by trenzterra** [comment]
Hi, my wordpress.org username is trenzterra as well.

By the way, is there any issue for merge if I had originally Googled for solutions and found this code on an article guide on a  website, and modified it to work in WordPress? Just so there won't be any copyright issue or what. Thanks.

**Comment by trenzterra** [comment]
It seems that I forgot to define 'event'. I have committed the change. Could you help me test it again on Firefox for Android?

**Comment by iamtakashi** [comment]
With the last change, it works in Firefox too.

**Comment by trenzterra** [comment]
Thanks. I've applied the changes.

**Pull request merged**